ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -228,6 +228,13 @@ interface {{ ethernet_interface }}
 {%         elif ethernet_interfaces[ethernet_interface].spanning_tree_bpdufilter is arista.avd.defined("disabled") %}
    spanning-tree bpdufilter disable
 {%         endif %}
+{%         if ethernet_interfaces[ethernet_interface].spanning_tree_guard is arista.avd.defined('loop') %}
```

Comment:
To avoid parsing all three `if`s for every interface I suggest:
```jinja
{% if ethernet_interfaces[ethernet_interface].spanning_tree_guard is arista.avd.defined %}
{%     if ethernet_interfaces[ethernet_interface].spanning_tree_guard == 'disabled' %}
   spanning-tree guard none
{%     else %}
   spanning-tree guard {{ ethernet_interfaces[ethernet_interface].spanning_tree_guard }}
{%     endif %}
{% endif %}
```

Reply:
added. Only drawback I see with this approach, is that it also allows/renders invalid options.. basically everything the user adds (besides "disabled")

Reply:
You are 100% correct. We do this almost everywhere else in AVD. We will address this issue later by introducing input validation with schemas in AVD 4.0.